snforge_std/src/cheatcodes/execution_info.cairo

```diff
@@ -0,0 +1,133 @@
+use starknet::{ContractAddress, testing::cheatcode, contract_address_const};
+use starknet::info::v2::ResourceBounds;
+use snforge_std::cheatcodes::CheatSpan;
+use super::super::_cheatcode::handle_cheatcode;
+
+#[derive(Serde, Drop, Copy)]
+pub struct CheatArguments<T> {
+    value: T,
+    span: CheatSpan,
+    target: ContractAddress,
+}
+
+#[derive(Serde, Drop, Copy)]
+pub enum Operation<T> {
+    StartGlobal: T,
+    Start: CheatArguments<T>,
+    Stop: ContractAddress,
+    StopGlobal,
+    Retain,
+}
```

Comment:
Why are some marked as pub while others are not?

Reply:
Probably because i copied them from rust code. Fixed